scripts/03-create-vscode-workspace.sh

```diff
@@ -158,23 +158,30 @@ cat << EOF
         "python.pythonPath": "$COMPOSE_HOME/etc/conda/envs/rapids/bin/python",
 
         "clangd.path": "/usr/bin/clangd",
```

Comment:
@trxcllnt Is there a reason we use `/usr/bin/clangd`? I would prefer `clangd` (fetch it from the `PATH`) since I don't have installation privileges to update `/usr/bin/clangd` on the lab machines and instead put in on my local path.
```suggestion
        "clangd.path": "clangd",
```

Reply:
Isn't this used inside the container though?

Reply:
That depends. I run VSCode over the SSH remote, and it runs `clangd` outside the container. I think that is the way most people use rapids-compose.

However, if you use VSCode's "Container" remote, then it would pull from within the container environment. Running `which clangd` inside the container currently points to the one in the conda environment [pinned to 11.1](https://github.com/rapidsai/cudf/blob/a9eb47cab6976e515af597cc6f9a90b846cb6706/conda/environments/cudf_dev_cuda11.5.yml#L12) (for clang-format). There is no `/usr/bin/clangd` in the container. Also, version 11.1 is too old to use with this PR, so we'd need to update the container's `clangd` using the LLVM apt repository if developers wish to use the "Container" remote.